eng/common/tools.ps1

```diff
@@ -365,8 +365,8 @@ function InitializeVisualStudioMSBuild([bool]$install, [object]$vsRequirements =
 
   # If the version of msbuild is going to be xcopied,
   # use this version. Version matches a package here:
-  # https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-eng&package=RoslynTools.MSBuild&protocolType=NuGet&version=17.2.1&view=overview
-  $defaultXCopyMSBuildVersion = '17.2.1'
+  # https://dev.azure.com/dnceng/public/_packaging?_a=package&feed=dotnet-eng&package=RoslynTools.MSBuild&protocolType=NuGet&version=17.3.1view=overview
```

Comment:
Are we missing an `&` between `17.3` and `view=`?

Reply:
Yes, though this is in comment in a file that is synced from the arcade repo, so it would need to be fixed there: https://github.com/dotnet/arcade/blob/release/7.0/eng/common/tools.ps1#L368  @mmitche